command/base.go

```diff
@@ -189,6 +195,31 @@ func (c *BaseCommand) Client() (*api.Client, error) {
 	return client, nil
 }
 
+func (c *BaseCommand) applyHCPConfig() {
+	hcpToken, err := hcpengine.GetHCPToken()
+	if err != nil {
+		c.UI.Error(err.Error())
```

Comment:
Why not return an error to the caller of Client and let them handle telling the user?